src/uct/api/uct.h

```diff
@@ -1175,6 +1230,38 @@ ucs_status_t uct_ep_connect_to_ep(uct_ep_h ep, const uct_device_addr_t *dev_addr
 
 
 /**
+ * @ingroup UCT_RESOURCE
+ * @brief Initiate a client-server connection to a remote peer.
+ *
+ * requires @ref UCT_IFACE_FLAG_CONNECT_TO_SOCKADDR capability.
```

Comment:
![uct_ep_create_sockaddr_doxygen](https://user-images.githubusercontent.com/2405183/29561242-75b9d084-873d-11e7-9c5a-dca8e03922f5.PNG)

@shamisp What do you mean? I attached a screenshot, can you take a look please?